CHANGELOG/CHANGELOG-2.4.md

```diff
@@ -0,0 +1,80 @@
+  - Use Protobuf message internally on FlowAggregator. ([#7253](https://github.com/antrea-io/antrea/pull/7253), [@antoninbas])
+  - Use Protobuf / gRPC between FlowExporter and FlowAggregator by default, and allow disabling IPFIX collector via `aggregatorTransportProtocol`. ([#7264](https://github.com/antrea-io/antrea/pull/7264), [@antoninbas])
+  - Push flow-aggregator image to `ghcr.io` registry. ([#7036](https://github.com/antrea-io/antrea/pull/7036), [@antoninbas])
+- Log error when OVS meter drops packets, which helps to evaluate whether increasing the packetInRate configuration is needed. ([#7242](https://github.com/antrea-io/antrea/pull/7242), [@tnqn])
+- Log PacketIn drops when dispatching to per-category queues to improve troubleshooting. ([#7174](https://github.com/antrea-io/antrea/pull/7174), [@tnqn])
+- Increase the default packet-in rate limit to 5000. ([#7243](https://github.com/antrea-io/antrea/pull/7243), [@tnqn])
+- Sync affected groups in the Antrea Controller when a Pod goes into `Terminated` state, to ensure that the Pod is excluded from NetworkPolicy source and destination immediately. ([#7217](https://github.com/antrea-io/antrea/pull/7217), [@Dyanngg])
+- Add more configuration values to the flow-aggregator chart. ([#7138](https://github.com/antrea-io/antrea/pull/7138), [@antoninbas])
+- Decouple sending of ICMP probes & latency reporting for NodeLatencyMonitor, which can improve accuracy of measurements and reduce system load. ([#7189](https://github.com/antrea-io/antrea/pull/7189), [@g4rud4kun])
+- Add ICMP Rule for NodeLatencyMonitor to make it work when the Node is configured with iptables default DROP policy. ([#7011](https://github.com/antrea-io/antrea/pull/7011), [@Dhruv-J])
+- Handle Pod UID updates in PodStore to handle a corner case that old and new Pods from update handler are actually different objects. ([#6964](https://github.com/antrea-io/antrea/pull/6964), [@antoninbas])
+- Support configuring file permission for the Antrea CNI configuration file. ([#7098](https://github.com/antrea-io/antrea/pull/7098), [@luolanzone])
+- Install iptables rules to allow WireGuard packets to ensure Antrea with WireGuard can work properly when the Node is configured with iptables default DROP policy. ([#7030](https://github.com/antrea-io/antrea/pull/7030), [@wenyingd])
+- Make IPPool `prefixLength` and `gateway` immutable. ([#7186](https://github.com/antrea-io/antrea/pull/7186), [@wenqiq])
+- Periodically sync permanent neighbors to ensure route correctness for Antrea host gateway interface. ([#7238](https://github.com/antrea-io/antrea/pull/7238), [@hongliangl])
+- Rename a SR-IOV VF device, which is configured as a secondary Pod interface, back to the original name when the Pod is deleted. ([#7144](https://github.com/antrea-io/antrea/pull/7144), [@luolanzone])
+- Update go-ipfix to v0.14.0. ([#7080](https://github.com/antrea-io/antrea/pull/7080), [@antoninbas])
```

Comment:
we are going to end up with go-ipfix v0.16.0 in this release, so that would be confusing

it was also meant as a bug fix for #7079, which maybe I should have made clearer

so maybe we could replace this line with "Fix invalid IPFIX UDP traffic fragmentation in the Flow Aggregator". You can keep the PR reference the same and move the entry to the Fixed` section.